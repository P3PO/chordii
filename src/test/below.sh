#!/bin/sh
echo a2crd - checking conversion of titles, chords below lyrics...
./a2crd test/data/below.input > below.actual
diff below.actual test/data/below.expected
exitcode=$?
rm below.actual
exit $exitcode
