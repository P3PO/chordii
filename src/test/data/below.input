Wonderful Time
Words and Music (C)2007 Adam Monsen

Everybody, let's have a wonderful time
C    G     F
Take it slowly, while you're feelin' fine
C       G       F
One strage day shouldn't take you by surprise
C          E7            F
But all right, hey hey, hey hey
C              G        F

Ahhh ahhh ahhh
C    Am   G
Ahhh ahhh ahhh
C    Am   G

Wonderful time
F         C
