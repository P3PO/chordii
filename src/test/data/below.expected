{title:Wonderful Time}
{subtitle:Words and Music (C)2007 Adam Monsen}

[C]Every[G]body, [F]let's have a wonderful time
[C]Take it [G]slowly, [F]while you're feelin' fine
[C]One strage [E7]day shouldn't [F]take you by surprise
[C]But all right, [G]hey hey, [F]hey hey

[C]Ahhh [Am]ahhh [G]ahhh
[C]Ahhh [Am]ahhh [G]ahhh

[F]Wonderful [C]time
